src/blocks/example.py

```diff
@@ -29,6 +30,22 @@ def get_info(self):
         }
 
     def execute(self, params):
+        if params['param1'] == 'suspend':
+            return self.suspend({'params': params, 'suspend_count': 1},
+                                {'param1': {'type': 'string', 'description': 'The first parameter.'}})
```

Comment:
I propose we rather let each object have an info dictionary such as the one returned from get_info. Then we can fetch the params type and description from there instead of hardcoding it into the response